src/App.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  BrowserRouter, Route, Routes, Navigate,
+} from 'react-router-dom';
+import './App.css';
+import Cookies from 'js-cookie';
+import Login from './components/Login';
+import Signup from './components/Signup';
+import Cars from './components/Cars';
+import ShowCar from './components/ShowBike';
+import AddReservation from './components/AddReservation';
+import AddCars from './components/AddCars';
+import Layout from './components/Layout';
+import ReservationList from './components/ReservationList';
+import DeleteCars from './components/DeleteCars';
+
+function AuthenticatedRoute({ element }) {
+  const isCookieSet = Cookies.get('Authorization') !== undefined;
+  return isCookieSet ? element : <Navigate to="/login" />;
+}
+function App() {
```

Comment:
- [ ]  Please, kindly refactor these functions to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```


```javascript
const AuthenticatedRoute = ({ element }) => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪